sns-book/src/domain-name/records/records-v2.md

```diff
@@ -24,7 +25,7 @@ const freshRecord = await verifyStaleness(connection, record, domain);
 
 ## Verify Right of Association
 
-You're also able to verify the authenticity of a record with Records V2 using the `verifyRightOfAssociation` function. The function returns a boolean indicating if the record is authentic or not. Unique to this function is the verifier parameter, which is the known public key of the on-chain/off-chain oracle used to verify authenticity of a record. This is currently supported for SOL, ETH, and URL records with support for further records on the way.
+You're also able to verify the authenticity of a record with Records V2 using the `verifyRightOfAssociation` function. The function returns a boolean indicating if the record is authentic or not. Unique to this function is the verifier parameter, which is the known public key of the on-chain/off-chain oracle used to verify authenticity of a record. This is currently supported for SOL, ETH, URL, and CNAME records with support for further records on the way.
```

Comment:
BNB and INJ are also supported

Reply:
Thank you, I've added them to the list and included them in the self signing section as well.